aii-ks/src/main/perl/ks.pm

```diff
@@ -1,9 +1,9 @@
 #${PMpre} NCM::Component::ks${PMpost}
 
 use EDG::WP4::CCM::Path qw (escape unescape);
-use NCM::Filesystem 18.12.0;
+use NCM::Filesystem 19.12.0;
```

Comment:
you will need to bump this, as it will only be released in next release. to get the unittests to pass, i think latest lease, with subminor 1 should work.